lizrd/scripts/grid_utils.py

```diff
@@ -27,6 +27,28 @@ def get_machine_backend() -> MachineBackend:
         return MachineBackend.LOCAL
 
 
+def get_common_directory(runner: MachineBackend) -> str:
+    if runner == MachineBackend.ATHENA:
+        return "/net/pr2/projects/plgrid/plggllmeffi"
+    elif runner == MachineBackend.IDEAS:
+        return "/raid/NFS_SHARE/llm-random"
+    elif runner == MachineBackend.ENTROPY_GPU:
+        return "/common/llm-random"
+    else:
+        return "~"
```

Comment:
Preferably os.getenv("HOME") for a couple of reasons xd

Reply:
fair point